packages/outline-core/src/outline-element/outline-element.a11y.css

```diff
@@ -1,4 +1,7 @@
-.sr-only {
+.sr-only,
+.visually-hidden,
+::slotted(.visually-hidden),
+[aria-hidden='true'] {
```

Comment:
```suggestion
::slotted(.visually-hidden) {
```

Reply:
visually-hidden is not the same as hidden from screen reader ([aria-hidden-"true"]).

Reply:
Fixed in latest update.  Just pulled this from a client project so wasn't sure.